src/Serializer/PayloadSerializer.php

```diff
@@ -152,6 +152,12 @@ private function serializeAsEvent(Event $event): string
             $result['spans'] = array_values(array_map([$this, 'serializeSpan'], $event->getSpans()));
         }
 
+        if ($stacktrace = $event->getStacktrace()) {
```

Comment:
Can you please avoid assigning the variable in the `if` statement and check for `null` explicitly?